src/core/Microsoft.PowerToys.Settings.UI/Views/KeyboardManagerPage.xaml

```diff
@@ -0,0 +1,244 @@
+                                    Grid.Column="0"
+                                    Width="70"
+                                    Height="35"
+                                    HorizontalAlignment="Left">
+                                <TextBlock  Grid.Column="0"
+                                            Style="{StaticResource BodyTextBlockStyle}"
+                                            VerticalAlignment="Center"
+                                            TextAlignment="Center"
+                                            Text="{Binding From}">
+                                </TextBlock>
+                            </Border>
+                            <Border Background="{ThemeResource SystemBaseLowColor}"
+                                    Grid.Column="1"
+                                    Width="70"
+                                    Height="35"
+                                    HorizontalAlignment="Left">
+                                <TextBlock Grid.Column="1" 
+                                           Style="{StaticResource BodyTextBlockStyle}"
+                                           VerticalAlignment="Center"
+                                           TextAlignment="Center"
+                                           Text="{Binding To}">
+                                </TextBlock>
+                            </Border>
+                        </Grid>
+                    </DataTemplate>
+                </ListView.ItemTemplate>
+            </ListView>
+            
+            <Grid Margin="{StaticResource SmallTopMargin}">
+                <TextBlock Text="Edit Shortcuts"
+                           Style="{StaticResource SubtitleTextBlockStyle}" />
+                <Button Content="Remap" 
+                        Margin="{StaticResource MediumRightMargin}"
+                        Style="{StaticResource AccentButtonStyle}" 
+                        HorizontalAlignment="Right"/>
+            </Grid>
+
+            <TextBlock Text="To redefine shortcuts, press the remap button"
+                       Margin="{StaticResource XSmallTopMargin}"/>
+
+            <StackPanel Margin="{StaticResource SmallLeftTopRightBottomMargin}"
+                        Orientation="Horizontal">
+                <TextBlock Text="Shortcuts for:"
+                           FontWeight="Bold">
+                </TextBlock>
+                <TextBlock Text="Config-1"
+                           Margin="{StaticResource XSmallLeftMargin}"
+                           FontWeight="ExtraLight" >
+                </TextBlock>
+            </StackPanel>
+
+            <StackPanel Margin="{StaticResource SmallLeftMargin}"
+                        Orientation="Horizontal">
+                <TextBlock Text="From:"
+                           Width="200">
+                </TextBlock>
+                <TextBlock Text="To:"
+                           Width="200" 
+                           FontWeight="ExtraLight" >
```

Comment:
```suggestion
                  >
```
Readability issues. Maybe we can make it (Semi)Bold to make it stand out more?

Reply:
Updated the FontWeight to be `SemiBold`